SampleApps/SPMTest/SPMTest.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -5,8 +5,8 @@
         "package": "Braintree",
         "repositoryURL": "https://github.com/braintree/braintree_ios",
         "state": {
-          "branch": "5.0-development",
-          "revision": "d40d70573735f3ca20a3c7728c1e295f5b404ed3",
+          "branch": "kount-spm",
+          "revision": "b7003f358eb5a0b31f3764076b99a3fbc4277b2c",
```

Comment:
Once this PR is merged, we might want to open another one where the SPMTest app uses master.

Reply:
For test apps it's also possible to use local package. Therefore you would always have the right dependency.

Reply:
Hello 👋 . Yes, this is definitely a feature we take advantage of when developing. However, we don't want to check in (to Git) a developers local URL path for the `repositoryURL` in the Package.resolved file.